packages/hydra-cli/src/templates/variants/variants.mst

```diff
@@ -53,6 +56,16 @@ export class {{name}} {
     {{camelName}}{{^required}}?{{/required}}{{#required}}!{{/required}}: typeof {{tsType}};
     {{/is.union}}
 
+    {{#is.entity}}
+        @Field(type => {{tsType}}, {
+            {{^required}}nullable: true,{{/required}}
```

Comment:
For now, let's keep this field optional no matter what comes from the input schema. We don't want the mapping author to fill this field since we ask him to fill only the `{{camelName}}Id: string`.